skfda/exploratory/visualization/_utils.py

```diff
@@ -160,8 +167,13 @@ def _set_figure_layout(fig=None, axes=None,
     return fig, axes
 
 
-def _set_figure_layout_for_fdata(fdata, fig=None, axes=None,
-                                 n_rows=None, n_cols=None):
+def _set_figure_layout_for_fdata(
+    fdata: T,
```

Comment:
`T` is only used here so it should not be a `TypeVar`. Also, the attributes `dim_domain` and `dim_codomain` are used, so the type must be `FData`.